resources/compass/templates/migrator-job.yaml

```diff
@@ -18,6 +18,19 @@ spec:
         spec:
             restartPolicy: Never
             containers:
+                {{if eq .Values.global.database.embedded.enabled false}}
+                - name: cloudsql-proxy
+                  image: gcr.io/cloudsql-docker/gce-proxy:1.11
+                  command:
+                  - /bin/sh
+                  args:
+                  - -c
+                  - '/cloud_sql_proxy -instances={{ .Values.global.database.managedGCP.instanceConnectionName }}=tcp:5432 -credential_file=/secrets/cloudsql-instance-credentials/credentials.json & sleep 30; exit'
```

Comment:
Why do we need to sleep for 30 seconds in parallel?

Reply:
This way, this container will exit running status after 30 seconds,  won't block the job mark itself completed.

Reply:
What if the schema migration takes longer than 30 seconds? The proxy will exit and then the migrator will fail - do I understand correctly?

If so, the solution doesn't seem to be right. Can we do it the same way as killing Istio sidecar? Please see my other comment.

Reply:
Thanks for the suggestion, has updated this PR with an workaround.